clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -42,6 +42,8 @@ const char *SYCL::Linker::constructLLVMSpirvCommand(Compilation &C,
   } else {
     CmdArgs.push_back("-spirv-max-version=1.1");
     CmdArgs.push_back("-spirv-ext=+all");
+    if (C.getArgs().hasArg(options::OPT_fsycl_esimd))
```

Comment:
Not in a scope of this patch, but I think we shall add another option to enable translation of unknown to the translator intrinsic, for example for users: https://github.com/intel/llvm/issues/2122